pkgs/servers/pulseaudio/default.nix

```diff
@@ -114,38 +163,32 @@ stdenv.mkDerivation rec {
   # after the seventh)
   NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-I/usr/include";
 
-  installFlags =
-    [ "sysconfdir=$(out)/etc"
-      "pulseconfdir=$(out)/etc/pulse"
-    ];
+  NIX_LDFLAGS = lib.optionalString stdenv.isDarwin "-framework CoreServices -framework Cocoa -framework AudioUnit";
 
   postInstall = lib.optionalString libOnly ''
     rm -rf $out/{bin,share,etc,lib/{pulse-*,systemd}}
-    sed 's|-lltdl|-L${libtool.lib}/lib -lltdl|' -i $out/lib/pulseaudio/libpulsecore-${version}.la
-  ''
-    + ''moveToOutput lib/cmake "$dev" '';
+    moveToOutput lib/cmake "$dev"
```

Comment:
Should not this also be done when `!libOnly`?

Reply:
did that without !libOnly